packages/framework/src/Component/ClassExtension/ClassExtensionRegistry.php

```diff
@@ -52,27 +52,31 @@ public function addExtendedService(string $parentClassName, string $childClassNa
     }
 
     /**
+     * Other classes that are not entities or registered in service extension map
+     * I.e. data objects and controllers (other class types can by added by modifying the finder if necessary)
+     *
      * @return string[]
      */
-    protected function getDataObjectExtensionMap(): array
+    protected function getOtherClassesExtensionMap(): array
     {
         $finder = Finder::create()
             ->files()
             ->ignoreUnreadableDirs()
             ->in($this->frameworkRootDir)
-            ->name('/.*Data\.php/');
+            ->exclude('tests/Unit/')
```

Comment:
Why not exclude whole `tests` directory? Or even better search only in `$this->frameworkRootDir . '/src'`?

Reply:
true that, makes sense :+1: fixed the commit message, amended the last commit, rebased and push-forced :innocent: